full-stack-tests/core/src/frontend/standalone/BriefcaseTxns.test.ts

```diff
@@ -0,0 +1,137 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import * as path from "path";
+import { Guid, OpenMode, ProcessDetector } from "@bentley/bentleyjs-core";
+import { IModelJson, LineSegment3d, Point3d, Transform, YawPitchRollAngles } from "@bentley/geometry-core";
+import { Code, IModelWriteRpcInterface } from "@bentley/imodeljs-common";
+import {
+  BriefcaseConnection, EditingFunctions, ElementEditor3d, IpcApp,
+} from "@bentley/imodeljs-frontend";
+import { ElectronApp } from "@bentley/electron-manager/lib/ElectronFrontend";
+
+describe("BriefcaseTxns", () => {
+  if (ProcessDetector.isElectronAppFrontend) {
+    let imodel: BriefcaseConnection;
+
+    before(async () => {
+      await ElectronApp.startup();
+    });
+
+    after(async () => {
+      await ElectronApp.shutdown();
+    });
+
+    beforeEach(async () => {
+      const filePath = path.join(process.env.IMODELJS_CORE_DIRNAME!, "core/backend/lib/test/assets/planprojection.bim");
```

Comment:
is this the new way to make writeable tests? Do we copy that directory every time we run the test? We need to make sure we don't use that file in 2 different tests?

Reply:
This is the way I did it in InteractiveEditingSession.test.ts, which I must have found in somebody else's test. It doesn't copy. It's surely wrong.

Reply:
do you want to copy the file before opening it?